README.md

```diff
@@ -1,5 +1,5 @@
 # VOKUtilities
-> Maintained by @bryanluby
+> Maintained by [@bryanluby](https://github.com/bryanluby)
```

Comment:
will make sure to follow this format in the future - will update some of the other repos to this as well 👍